src/api/models/iuf/iuf-manifest-schema.yaml

```
@@ -34,7 +34,7 @@ description: >
   of the product release distribution in order for it to be discovered by the
   IUF.
 # This is the version of the IUF manifest schema itself
-version: 0.6.0
+version: 0.7.0
```

Since this is a backwards-compatible change, can you just bump the patch version of the schema. When the major version is zero, we use the minor version like the major version to denote backwards-incompatible changes.

```suggestion
version: 0.6.1
```

This is sorta important because if somebody says they're compatible with `^0.6.0`, we'd like them to work with this version. Having said that we don't really check this against this today anyway.

More info here: https://connect.us.cray.com/confluence/display/SHASTA/IUF+Adoption+Guide#IUFAdoptionGuide-CompatibleIUFVersion